lib/controllers/ServiceFabrikApiController.js

```diff
@@ -457,10 +472,11 @@ class ServiceFabrikApiController extends BaseController {
       .omit('repeatInterval')
       .set('instance_id', req.params.instance_id)
       .set('trigger', CONST.BACKUP.TRIGGER.SCHEDULED)
-      .set('space_guid', req.entity.space_guid)
+      .set('tenant_id', req.entity.tenant_id)
+      .set('plan_id', req.manager.plan.id)
       .set('service_id', req.manager.service.id)
       .value();
-    return this.cloudController.getOrgAndSpaceDetails(data.instance_id, data.space_guid)
+    return this.cloudController.getOrgAndSpaceDetails(data.instance_id, data.tenant_id) // Why all these needed ?
```

Comment:
can you remove the comment `Why all these needed ?`

Reply:
done